haskell-org-committee.markdown

```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
   * Rebecca Skinner (term ends 2022)
   * Alexandre Garcia de Oliveira (term ends 2022)
 
-In November of each year—or if a member steps down before their term is over—the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
-
 Committee members do not need to be deep technical Haskell experts.  Instead we look for enthusiasm towards improving the Haskell community.  We aim to represent the different facets of the community.  We aim to be diverse in terms of industry or research, and in terms of gender, race and location.
 
+### Process
+
+* The committee's governing rules are laid out in the [Board Resolution](https://wiki.haskell.org/wikiupload/9/9c/Haskell-org-board-res.pdf) of that month.
+* The committee uses the [haskell-community][list] for public discussion, where all interested parties may participate.
+* Proposals and meeting notes are stored in a [public repository][repo].
+* The committee will seek to reach consensus among its membership on its decisions. Ultimately, the committee will make decisions by more than half of the membership voting for a particular outcome.
+* These rules of operation may also be changed in the same way.
+
+#### Committee Membership
+
+The committee consists of 7 members. Members are expected to serve a 3 year term, and terms are staggered so that 2 or 3 members step down each year, at the end of October.
+
+The members will elect one of their number to be chair each year. The chair is responsible for making sure that things keep moving, and to ensure that a conclusion is reached on any issues raised.
+
+In November of each year — or if a member steps down before their term is over — the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
+
 After nominations have been solicited, the current committee—including any outgoing members—selects new members with a simple majority vote.
```

Comment:
```suggestion
After nominations have been solicited, the current committee—including any outgoing members not seeking reelection—selects new members with a simple majority vote.
```

The Wiki page is right: any members who renominated themselves after their term is up don't vote that year. The current page is missing that detail.